lib/rules/font-family-name-quotes/__tests__/index.js

```diff
@@ -400,36 +436,26 @@ testRule({
 			code: 'a { font-family: "Lucida Grande", "Arial", "sans-serif"; }',
 			fixed: 'a { font-family: "Lucida Grande", "Arial", sans-serif; }',
 			message: messages.rejected('sans-serif'),
-			line: 1,
-			column: 45,
```

Comment:
[note] I've removed checking a position from `fix: true` test cases since they are duplicated with the *non-fix* test cases above. No problems?